### PR TITLE
Update

### PR DESCRIPTION
**Pull request by AndreaLBarr**
update to app guard table

### CODE REVIEW

File: windows/security/threat-protection/microsoft-defender-application-guard/configure-md-app-guard.md [hunk 1]
```diff
@@ -58,8 +57,10 @@ These settings, located at **Computer Configuration\Administrative Templates\Win
...
-|Allow files to download to host operating system|Windows 10 Enterprise, 1803 or higher|Determines whether to save downloaded files to the host operating system from the Microsoft Defender Application Guard container.|**Enabled.** Allows users to save downloaded files from the Microsoft Defender Application Guard container to the host operating system.<br><br>**Disabled or not configured.** Users are not able to saved downloaded files from Application Guard to the host operating system.|
+|Allow files to download to host operating system|Windows 10 Enterprise, 1803 or higher|Determines whether to save downloaded files to the host operating system from the Microsoft Defender Application Guard container.|**Enabled.** Allows users to save downloaded files from the Microsoft Defender Application Guard container to the host operating system.Note that this creates a Share between the Host and Container that also allows for uploads from the host to the Application Guard container.<br><br>**Disabled or not configured.** Users are not able to save downloaded files from Application Guard to the host operating system.|
```
Comment (illfated) [inline windows/security/threat-protection/microsoft-defender-application-guard/configure-md-app-guard.md:60]:
```suggestion
|Allow files to download to host operating system|Windows 10 Enterprise, 1803 or higher|Determines whether to save downloaded files to the host operating system from the Microsoft Defender Application Guard container.|**Enabled.** Allows users to save downloaded files from the Microsoft Defender Application Guard container to the host operating system. Note that this creates a Share between the Host and Container that also allows for uploads from the host to the Application Guard container.<br><br>**Disabled or not configured.** Users are not able to save downloaded files from Application Guard to the host operating system.|
```
(sentence spacing issue only)

**Review by illfated — APPROVED** [review]
Rich Diff view: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9503/files?short_path=88df1bf#diff-88df1bf2fc8867f8ac97679535661842e38ac1573aee8ef90f211dae6f2ace4d

Looks good to me.

**Comment by AndreaLBarr** [comment]
@illfated Since this looks good could you please merge this since I don't have write permissions for this repo?

**Comment by illfated** [comment]
Wish I could, but unfortunately I am not a member of this organization. ❤️ Thank you for having faith in me that I would be, though.

@denisebmsft : For your information.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by denisebmsft — APPROVED** [review]
Thank you @AndreaLBarr

**Pull request merged**